src/sage/combinat/root_system/root_lattice_realization_algebras.py

```diff
@@ -566,13 +566,13 @@ def demazure_lusztig_operators(self, q1, q2, convention="antidominant"):
 
                 sage: K = QQ['q1,q2']
                 sage: q1, q2 = K.gens()
-                sage: for cartan_type in CartanType.samples(crystallographic=True): # long time 12s
+                sage: for cartan_type in CartanType.samples(crystallographic=True): # long time (12s)
```

Comment:
```suggestion
                sage: for cartan_type in CartanType.samples(crystallographic=True):  # long time (12s)
```
separate with 2 spaces before `#` please